test/runtests.jl

```diff
@@ -459,4 +459,34 @@ using Test
         @test length(net.buses) == 2
         @test length(net.loads) == 1
     end
+
+    @testset "empty network" begin
+
+        empty_net = Network(
+            30,
+            CaseID(),
+            Buses30(),
+            Loads(),
+            FixedShunts(),
```

Comment:
technically a v30 file doesn't have `FixedShunts`, so for the sake of making tests an accurate example:
```suggestion
            nothing,
```